src/templates/storage-list.html

```
@@ -13,6 +13,18 @@
   <!-- [html-validate-disable-block prefer-native-element: suppress errors] -->
 {%- endif %}
 <div id='storageDiv'>
+  {# stepper used in the entity-create flow to pick how many containers go in this location #}
+  {% macro render_stepper(storage_id) %}
+    <div class='input-group input-group-sm mt-1' style='max-width:160px' data-storage-id='{{ storage_id }}'>
+      <div class='input-group-prepend'>
+        <button type='button' class='btn btn-secondary' data-action='container-qty-minus' data-storage-id='{{ storage_id }}' aria-label='{{ 'Decrease'|trans }}'><i class='fas fa-minus'></i></button>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)